library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/recorder/mapper/BaseTextViewWireframeMapperTest.kt

```diff
@@ -0,0 +1,213 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.recorder.mapper
+
+import android.graphics.Typeface
+import android.graphics.drawable.ColorDrawable
+import android.widget.TextView
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.datadog.android.sessionreplay.recorder.aMockView
+import com.datadog.android.sessionreplay.recorder.densityNormalized
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import fr.xgouchet.elmyr.Forge
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+
+internal abstract class BaseTextViewWireframeMapperTest : BaseWireframeMapperTest() {
+
+    lateinit var testedTextWireframeMapper: TextWireframeMapper
+
+    @BeforeEach
+    fun `set up`() {
+        testedTextWireframeMapper = initTestedMapper()
+    }
+
+    protected open fun initTestedMapper(): TextWireframeMapper {
+        return TextWireframeMapper()
+    }
+
+    @ParameterizedTest
+    @MethodSource("textTypefaces")
+    fun `M resolve a TextWireframe W map() { TextView with fontStyle }`(
+        typeface: Typeface,
```

Comment:
minor: let's rename to `fakeTypeface` for the consistency and avoid a possible confusion when `this` is not specified explicitly.